lib/setup.js

```diff
@@ -1,22 +1,67 @@
 const path = require('path');
+const fs = require('fs');
+const vite = require('./vite.js');
+const getPort = require('get-port')
 
 const vite_dev_server = {
-  port: 5174
+  port: process.argv.includes('--test-worker') ? null : 5173
 };
 try {
   const nightwatch_config = require(path.resolve('./nightwatch.conf.js'));
   Object.assign(vite_dev_server, nightwatch_config.vite_dev_server || {});
 } catch (err) {}
 
-const vite = require('./vite.js');
+const projectJsConfigFile = path.join(process.cwd(), 'vite.config.js');
+const projectTsConfigFile = path.join(process.cwd(), 'vite.config.ts');
+
+const hasProjectJsConfigFile = () => {
+  try {
+    if(fs.statSync(projectJsConfigFile).isFile()) {
+      return projectJsConfigFile;
+    }
```

Comment:
I think the TS file should be given preference. However, going with the future, we will adapt typescript so that the typescript first approach would be more helpful.

Reply:
adopting TS is a distant future.